src/sdk/types.ts

```diff
@@ -212,7 +220,25 @@ export type Category =
 
 export type PlatformDisplayConfig = {
   name: string;
-  iconURI: string;
+  /** @deprecated */
+  iconURI?: string;
+  logos: {
+    defaultURI: string;
```

Comment:
Can you also provide a note for the `defaultURI` as well?

Mainly want to make sure people don't get confused on the differences and how each of them are used